validators/ro.go

```diff
@@ -28,27 +28,34 @@ func (v Rondonia) IsValid(insc string) bool {
 
 	rule := v.getRule()
 
-	if rule.IsCorrectSize(insc, 9) {
+	switch {
+	case rule.IsCorrectSize(insc, 9):
+		{
 
-		base := insc[3:8]
-		weights := rule.GetWeight(6, 5)
-		total := rule.CalculateTotal(base, 5, weights)
-		var digit = rule.GetDigit(total, moduleDivisor)
+			base := insc[3:8]
```

Comment:
Fala @darllantissei blz?
Obrigado pela contribuição amigo... 👍🏻 

Mas consegue dar uma olhada porque essa formatação de código mudou? Qual IDE vc está usando? VSCode, Goland?
Tem usado alguma tool pra isso?

Pode ser que na época eu tenho feito errado também hehe

Reply:
Opa @dilowagner tudo certo!? 

Então... eu uso o VScode com a extensão Go For VSCode(https://code.visualstudio.com/docs/languages/go) que atualmente está na versão v0.39.1.